students/scott_bromley/lesson03/mailroom.py

```diff
@@ -16,45 +16,38 @@
 
 
 def main():
-    menu = ("Send a Thank You", "Create a Report", "quit")
+    menu = ("1 - Send a Thank You", "2 - Create a Report", "3 - quit")
     while True:
-        response = get_user_input(f"Please enter an option from the following menu: \n {menu} > ")
-        if response.lower() == "send a thank you":
+        response = int(get_user_input(f"Please enter an option from the following menu: \n {menu} > "))
+        if response == 1:
             send_thank_you()
-        elif response.lower() == "create a report":
+        elif response == 2:
             create_report()
-        elif response.lower() == "quit":
+        elif response == 3:
             break
         else:
             print(f"The option you entered '{response}' is invalid, please try again.")
-            continue
 
 
 def send_thank_you():
-    '''
+    """
     prompts user for a donor name; if name exists, prompts for donation, otherwise adds to donors
     list displays a list of donors
     generates
     :return: None
-    '''
-    full_name = get_user_input("Please enter the name of a donor: > ").title()
-    for donor in donors:
-        if full_name in donors:
-            donation = float(get_user_input("Please enter the donation amount: $").strip('$'))
-            add_donation(full_name, donation)
-            format_thank_you(full_name, donation)
-            break
-        elif full_name.lower() == "list":
-            print(f"List of donors: {', '.join(get_donors(donors))}")
-            break
+    """
+    donor_name = get_user_input("Please enter the name of a donor: > ")
+    if donor_name == 'list':
+        print(f"List of donors: {[', '.join(donor[0]) for donor in donors]}")
+    else:
+        donation = float(get_user_input("Please enter the donation amount: $").strip('$'))
+        for donor in donors:
+            if donor[0] == donor_name:
+                donor[1].append(donation)
```

Comment:
this is minor - but you would typically put a break here to stop looping once u found your donor.